flash_examples/finetuning/graph_classification.py

```diff
@@ -0,0 +1,36 @@
+import flash
+from flash.core.data.data_source import DefaultDataKeys
+from flash.core.data.transforms import ApplyToKeys
+from flash.core.utilities.imports import _PYTORCH_GEOMETRIC_AVAILABLE
+from flash.graph.classification.data import GraphClassificationData
+from flash.graph.classification.model import GraphClassifier
+
+if _PYTORCH_GEOMETRIC_AVAILABLE:
+    import torch_geometric.transforms as T
+    from torch_geometric.datasets import TUDataset
+else:
+    raise ModuleNotFoundError("Please, pip install -e '.[graph]'")
+
+dataset = TUDataset("data", name='IMDB-BINARY')
```

Comment:
It might be a good idea to shuffle the dataset before creating train/test splits, e.g.:
```python
dataset = TUDataset("data", name='IMDB-BINARY').shuffle()
```